akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -183,7 +186,7 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
   /** INTERNAL API. */
   override protected[akka] def aroundPreRestart(reason: Throwable, message: Option[Any]): Unit = {
     try {
-      internalStash.unstashAll()
+      unstashInternally(all = true)
```

Comment:
If the ``internalStash.unstash/all`` method has been encapsulated, perhaps call it directly in other places would not be a good idea for me.